### ISSUE TITLE
I wasn't there!

### ISSUE THREAD

**Issue by Grantb2** [petewarden/iPhoneTracker#20]
Any idea how to interpret info that says I was someplace that I have not been with my iPhone?

Specifically, I live in San Diego, but the tracker says I was all over northern Las Vegas in late June of 2010, shortly after getting my phone. The last time I was in Vegas, however, is at least 3 years ago. It was among one of the first iPhone 4's shipped out, so no way it was a return that had previously been in that area.

**Comment by jayavant** [comment]
I had to laugh at this. My track shows me in a part of Australia I have never visited, while my phone and I were in Korea.

**Comment by Grantb2** [comment]
Didn't realize that iPhones can swim.

**Comment by tmcw** [comment]
One of the theories is that iPhones have final assembly in Las Vegas and are thus powered on at that point. Might hold true for other locations.

**Comment by petewarden** [comment]
We are seeing a lot of people with Vegas on their phones. Unless there were
a lot of lost weekends, we've wondered about either an error in the cell
data or an Apple facility there.

**Comment by marookkenja** [comment]
Also: Note that the points is the GSM antenna/wifi hotspot location, NOT the phone location!!!! And the iPhone might be able to see/log GSM antennas that are up to 100 MILES away! Does that make it more valid?

**Comment by Grantb2** [comment]
Even if my phone were logging antennae 100 miles away on a regular basis, the Las Vegas locations would make no sense.

**Comment by yahelc** [comment]
Yeah, I haven't been within 2500 miles of Las Vegas with my iPhone and I too got this weird Vegas data point.
